### PR TITLE
Brooklyn Feature: Point To Site IPSec custom policy set on Gateway

### PR DESCRIPTION
**Pull request by Nilambari**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by MiYanni** [comment]
Cmdlet review here: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/45

**Comment by cormacpayne** [comment]
@Nilambari please take a look at the build failures in the markdown help files:

| Target | Description |
| -- | -- |
| New-AzureRmVirtualNetworkGateway.md | Trailing yaml string found in description for   parameter VpnClientIpsecPolicy. Please move the trailing yaml string to a line by itself. |
| Set-AzureRmVirtualNetworkGateway.md | Trailing yaml string found in description for parameter   VpnClientIpsecPolicy. Please move the trailing yaml string to a line by itself. |

File: src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj [hunk 1]
```diff
@@ -557,6 +557,9 @@
     <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.VirtualNetworkGatewayConnectionTests\TestVirtualNetworkGatewayConnectionwithIpsecPoliciesCRUD.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.VirtualNetworkGatewayConnectionTests\TestVirtualNetworkGatewayP2SAndSKU.json">
```
Comment (MiYanni) [inline src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj]: The path is wrong here. It should be:
`SessionRecords\Commands.Network.Test.ScenarioTests.VirtualNetworkGatewayTests\TestVirtualNetworkGatewayP2SAndSKU.json`
The word `Connection` isn't in the folder path for this file.

**Comment by MiYanni** [comment]
@Nilambari I added a comment that shows the current issue. The path for a json file is incorrect. Can you please fix it?

**Comment by Nilambari** [comment]
@cormacpayne @MiYanni Incorporated code review comments and fix build issues. Also, enabled skipped Brooklyn tests and added their latest recordings.  Can you Please help accept the PR? Thanks!

**Comment by MiYanni** [comment]
@Nilambari It looks like your help file has an issue.
```powershell
  PS C:\> New-AzureRmVirtualNetworkGateway -Res...'
   Expect Heading
  Exception calling "NodeModelToMamlModel" with "1" argument(s): "C:\workspace\powershell\src\Package\Debug\ResourceManag
  er\AzureResourceManager\AzureRM.Network\help\New-AzureRmVpnClientIpsecPolicy.md:26:(604) '```
  PS C:\> New-AzureRmVirtualNetworkGateway -Res...'
   Expect Heading"
  At C:\Program Files\WindowsPowerShell\Modules\platyPS\0.8.1\platyPS.psm1:1372 char:9
  +         $maml = $t.NodeModelToMamlModel($model)
  +         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
      + FullyQualifiedErrorId : HelpSchemaException
```

**Comment by Nilambari** [comment]
@MiYanni Fixed the help file error.

**Comment by MiYanni** [comment]
@Nilambari You have static analysis errors. It looks like multiple things have plural names. Were these things you added? If so, they'll need to be singular. File here (look at everything marked as `"1"` severity):
https://azuresdkci.westus2.cloudapp.azure.com/job/powershell/4481/artifact/src/Package/SignatureIssues.csv/*view*/

This should have been mentioned in the design review.

**Comment by Nilambari** [comment]
@MiYanni Can you Please tell me what needs to be fixed in order for build to pass?

**Comment by MiYanni** [comment]
@azuresdkci Retest this please.

**Comment by Nilambari** [comment]
Build is passed. Can you help merge the PR? Thanks!

File: src/ResourceManager/Dns/Commands.Dns.Test/ScenarioTests/ZoneTests.cs [hunk 2]
```diff
@@ -31,36 +31,31 @@ public void TestZoneCrud()
             DnsTestsBase.NewInstance.RunPowerShellTest("Test-ZoneCrud");
         }
 
-        [Fact]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Fact(Skip = "Test is failing, service team needs to re-record")]
```
Comment (cormacpayne) [inline src/ResourceManager/Dns/Commands.Dns.Test/ScenarioTests/ZoneTests.cs]: @Nilambari why are these tests failing? @muwaqar would you mind working with Nilambari to enable these tests?

File: src/ResourceManager/Network/ChangeLog.md [hunk 3]
```diff
@@ -34,6 +34,16 @@
 * Add support for DDoS protection plan resource
 * Introduced multiple breaking changes
     - Please refer to the migration guide for more information
+* Updated below commands for feature: Point to Site IPsec custom policy set/remove on Brooklyn Gateway.
```
Comment (cormacpayne) [inline src/ResourceManager/Network/ChangeLog.md]: @Nilambari please move the snippet you are adding to below the `## Current Release` header

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/GetAzureVpnClientIpsecParametersCommand.cs [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            HelpMessage = "The resource group name.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public virtual string ResourceGroupName { get; set; }
+
+        public override void Execute()
+        {
+            base.Execute();
+
+            if (ShouldProcess("AzureRmVpnClientIpsecParameter", VerbsCommon.Get))
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/GetAzureVpnClientIpsecParametersCommand.cs]: @Nilambari please remove the call to `ShouldProcess` for this cmdlet

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureRmVpnClientIpsecParametersCommand.cs [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+            MNM.PfsGroup.PFS2,
+            MNM.PfsGroup.None,
+            IgnoreCase = false)]
+        public string PfsGroup { get; set; }
+
+        public override void Execute()
+        {
+            base.Execute();
+
+            if (ShouldProcess(Properties.Resources.CreatingResourceMessage + Properties.Resources.VirtualNetworkGatewayVpnClientIpsecPolicy))
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureRmVpnClientIpsecParametersCommand.cs]: @Nilambari it doesn't appear that this cmdlet doesn't make any calls to the server, it's just creating an in-memory object. If that is the case, then you don't need `ShouldProcess` and you can remove `SupportsShouldProcess` from the cmdlet attribute

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureRmVpnClientIpsecParametersCommand.cs [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The Vpnclient IPSec encryption algorithm (IKE Phase 1)")]
+        [ValidateNotNullOrEmpty]
+        [ValidateSet(
+            MNM.IpsecEncryption.GCMAES256,
+            MNM.IpsecEncryption.GCMAES128,
+            MNM.IpsecEncryption.AES256,
+            MNM.IpsecEncryption.AES128,
+            IgnoreCase = false)]
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureRmVpnClientIpsecParametersCommand.cs]: @Nilambari is there any reason we need to be case sensitive for all of these parameters using `ValidateSet`? I think we should make all `ValidateSet` attributes case insensitive, so removing the `IgnoreCase = False` line from each one will resolve this issue.

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureRmVpnClientIpsecPolicyCommand.cs [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+            MNM.PfsGroup.PFS2,
+            MNM.PfsGroup.None,
+            IgnoreCase = false)]
+        public string PfsGroup { get; set; }
+
+        public override void Execute()
+        {
+            base.Execute();
+
+            if (ShouldProcess(Properties.Resources.CreatingResourceMessage + Properties.Resources.VirtualNetworkGatewayVpnClientIpsecPolicy))
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureRmVpnClientIpsecPolicyCommand.cs]: @Nilambari same comment here about removing `ShouldProcess` if this is only creating an in-memory object

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/RemoveAzureRmVpnClientIpsecParametersCommand.cs [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Azure.Management.Network;
+using System;
+using System.Management.Automation;
+using MNM = Microsoft.Azure.Management.Network.Models;
+using Microsoft.Azure.Commands.Network.VirtualNetworkGateway;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.Remove, "AzureRmVpnClientIpsecParameter", DefaultParameterSetName = "ByFactoryName", SupportsShouldProcess = true)]
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/RemoveAzureRmVpnClientIpsecParametersCommand.cs]: @Nilambari please add an `OutputType` to this cmdlet (it should be `bool`)

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/RemoveAzureRmVpnClientIpsecParametersCommand.cs [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+    public class RemoveAzureVpnClientIpsecParametersCommand : VirtualNetworkGatewayBaseCmdlet
+    {
+        [Parameter(
+            ParameterSetName = ParameterSetNames.ByFactoryName,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/RemoveAzureRmVpnClientIpsecParametersCommand.cs]: @Nilambari you can remove the `ValueFromPipelineByPropertyName` property from all parameters that aren't `ResourceId`

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/RemoveAzureRmVpnClientIpsecParametersCommand.cs [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+            HelpMessage = "The Azure resource ID.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Don't ask for confirmation.")]
+        public SwitchParameter Force { get; set; }
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/RemoveAzureRmVpnClientIpsecParametersCommand.cs]: @Nilambari the `Force` parameter should only be used in cmdlets that require additional prompting outside of the typical `ShouldProcess` prompt. If this cmdlet does not prompt the user outside of that, then you should remove it

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/SetAzureVpnClientIpsecParametersCommand.cs [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+using MNM = Microsoft.Azure.Management.Network.Models;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.Set,
+         "AzureRmVpnClientIpsecParameter",
+         DefaultParameterSetName = VirtualNetworkGatewayParameterSets.Default,
+         SupportsShouldProcess = true),
+     OutputType(typeof(PSVpnClientIPsecParameters))]
+    public class SetAzureVpnClientIpsecParametersCommand : VirtualNetworkGatewayBaseCmdlet
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/SetAzureVpnClientIpsecParametersCommand.cs]: @Nilambari this cmdlet should allow users to provide an `InputObject` or `ResourceId` parameter (similar to what was done for `Remove-*`)

File: src/ResourceManager/Network/Commands.Network/help/Get-AzureRmVpnClientIpsecParameter.md [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+## DESCRIPTION
+The Virtual Network Gateway is the object representing your gateway in Azure.
+
+The **Get-AzureRmVpnClientIpsecParameter** cmdlet returns the object of your vpn ipsec parameters set on gateway in Azure based on Gateway Name and Resource Group Name.
+
+## EXAMPLES
+
+### 1: Gets the vpn Ipsec parameters set on Virtual Network Gateway for Point to site connections.
+```
+Get-AzureRmVpnClientIpsecParameter -Name myGateway -ResourceGroupName myRG
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/help/Get-AzureRmVpnClientIpsecParameter.md]: @Nilambari please add the corresponding output for when this cmdlet is run

File: src/ResourceManager/Network/Commands.Network/help/New-AzureRmVpnClientIpsecParameter.md [hunk 13]
```diff
@@ -0,0 +1,219 @@
...
+```
+
+## DESCRIPTION
+This command allows the users to create the Vpn ipsec parameters object specifying one or all values such as IpsecEncryption,IpsecIntegrity,IkeEncryption,IkeIntegrity,DhGroup,PfsGroup to set on the existing VPN gateway.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> $vpnclientipsecparams1 = New-AzureRmVpnClientIpsecParameter -IpsecEncryption AES256 -IpsecIntegrity SHA256 -SALifeTime 86473 -SADataSize 429498 -IkeEncryption AES256 -IkeIntegrity SHA384 -DhGroup DHGroup2 -PfsGroup PFS2
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/help/New-AzureRmVpnClientIpsecParameter.md]: @Nilambari same comment about adding corresponding output

File: src/ResourceManager/Network/Commands.Network/help/New-AzureRmVpnClientIpsecPolicy.md [hunk 14]
```diff
@@ -0,0 +1,230 @@
...
+### Define vpn ipsec policy object:
+```
+PS C:\>$vpnclientipsecpolicy = New-AzureRmVpnClientIpsecPolicy -IpsecEncryption AES256 -IpsecIntegrity SHA256 -SALifeTime 86472 -SADataSize 429497 -IkeEncryption AES256 -IkeIntegrity SHA256 -DhGroup DHGroup2 -PfsGroup None
+```
+
+This cmdlet is used to create the vpn ipsec policy object using the passed one or all parameters' values which user can pass to param:VpnClientIpsecPolicy of PS command let: New-AzureRmVirtualNetworkGateway (New VPN Gateway creation) / Set-AzureRmVirtualNetworkGateway (existing VPN Gateway update) in ResourceGroup :
+
+### Create new virtual network gateway with setting vpn custom ipsec policy:
+```
+PS C:\> New-AzureRmVirtualNetworkGateway -ResourceGroupName vnet-gateway -name myNGW -location $location -IpConfigurations $vnetIpConfig -GatewayType Vpn -VpnType RouteBased -GatewaySku VpnGw1 -VpnClientIpsecPolicy $vpnclientipsecpolicy
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/help/New-AzureRmVpnClientIpsecPolicy.md]: @Nilambari same comment about adding corresponding output

File: src/ResourceManager/Network/Commands.Network/help/New-AzureRmVpnClientIpsecPolicy.md [hunk 15]
```diff
@@ -0,0 +1,230 @@
...
+```
+
+### Get virtual network gateway to see if vpn custom policy is set correctly:
+```
+$gateway = Get-AzureRmVirtualNetworkGateway -ResourceGroupName vnet-gateway -name myNGW
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/help/New-AzureRmVpnClientIpsecPolicy.md]: @Nilambari nit: please add the `PS C:\>` string before this command to be consistent with the line below it

File: src/ResourceManager/Network/Commands.Network/help/Remove-AzureRmVpnClientIpsecParameter.md [hunk 16]
```diff
@@ -0,0 +1,186 @@
...
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/remove-azurermvpnclientipsecparameter
+schema: 2.0.0
+---
+
+# Remove-AzureRmVpnClientIpsecParameter
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/help/Remove-AzureRmVpnClientIpsecParameter.md]: @Nilambari please fill in a synopsis

File: src/ResourceManager/Network/Commands.Network/help/Remove-AzureRmVpnClientIpsecParameter.md [hunk 17]
```diff
@@ -0,0 +1,186 @@
...
+## DESCRIPTION
+The Virtual Network Gateway is the object representing your gateway in Azure.
+
+The **Remove-AzureRmVpnClientIpsecParameter** cmdlet removes the vpn custom ipsec parameters set on your Virtual Network Gateway, which in turn sets default vpn ipsec policy on VPN gateway based on VirtualNetworkGateway Name and Resource Group Name passed.
+
+## EXAMPLES
+
+### 1: Deletes the set vpn ipsec parameters set on the Virtual Network Gateway
+```
+Remove-AzureRmVpnClientIpsecParameter -VirtualNetworkGatewayName myGateway -ResourceGroupName myRG
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/help/Remove-AzureRmVpnClientIpsecParameter.md]: @Nilambari same comment about adding the corresponding output

File: src/ResourceManager/Network/Commands.Network/help/Set-AzureRmVpnClientIpsecParameter.md [hunk 18]
```diff
@@ -0,0 +1,145 @@
...
+## DESCRIPTION
+The **Set-AzureRmVpnClientIpsecParameter** cmdlet sets the vpn ipsec parameters for existing virtual network gateway.
+When Virtual network gateway is created, it sets the set of default vpn ipsec policies on Gateway. In case, Point to site user wants to use certain custom ipsec policy to connect to VPN Gateway, user has to set that ipsec policy on VPN Gateway first. **Set-AzureRmVpnClientIpsecParameter** provides a way to do that.
+
+## EXAMPLES
+
+### Example 1 : Sets a custom vpn ipsec policy to existing virtual network gateway.
+```powershell
+PS C:\>$vpnclientipsecparams = New-AzureRmVpnClientIpsecParameter -IpsecEncryption AES256 -IpsecIntegrity SHA256 -SALifeTime 86473 -SADataSize 429498 -IkeEncryption AES256 -IkeIntegrity SHA384 -DhGroup DHGroup2 -PfsGroup PFS2
+PS C:\> $setvpnIpsecParams = Set-AzureRmVpnClientIpsecParameter -VirtualNetworkGatewayName "ContosoLocalGateway" -ResourceGroupName "ContosoResourceGroup" -VpnClientIPsecParameter $vpnclientipsecparams
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/help/Set-AzureRmVpnClientIpsecParameter.md]: @Nilambari same comment about adding corresponding output

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureRmVpnClientIpsecPolicyCommand.cs (hunk 7 above)
Comment (Nilambari) [reply src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureRmVpnClientIpsecPolicyCommand.cs]: Done.

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/RemoveAzureRmVpnClientIpsecParametersCommand.cs (hunk 9 above)
Comment (Nilambari) [reply src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/RemoveAzureRmVpnClientIpsecParametersCommand.cs]: Done.

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/RemoveAzureRmVpnClientIpsecParametersCommand.cs (hunk 10 above)
Comment (Nilambari) [reply src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/RemoveAzureRmVpnClientIpsecParametersCommand.cs]: Done.

File: src/ResourceManager/Network/Commands.Network/help/New-AzureRmVpnClientIpsecPolicy.md (hunk 14 above)
Comment (Nilambari) [reply src/ResourceManager/Network/Commands.Network/help/New-AzureRmVpnClientIpsecPolicy.md]: Done.

File: src/ResourceManager/Network/Commands.Network/help/New-AzureRmVpnClientIpsecPolicy.md (hunk 15 above)
Comment (Nilambari) [reply src/ResourceManager/Network/Commands.Network/help/New-AzureRmVpnClientIpsecPolicy.md]: Done.

File: src/ResourceManager/Network/Commands.Network/help/Remove-AzureRmVpnClientIpsecParameter.md (hunk 16 above)
Comment (Nilambari) [reply src/ResourceManager/Network/Commands.Network/help/Remove-AzureRmVpnClientIpsecParameter.md]: Done.

File: src/ResourceManager/Network/Commands.Network/help/Remove-AzureRmVpnClientIpsecParameter.md (hunk 17 above)
Comment (Nilambari) [reply src/ResourceManager/Network/Commands.Network/help/Remove-AzureRmVpnClientIpsecParameter.md]: Done.

File: src/ResourceManager/Network/Commands.Network/help/Set-AzureRmVpnClientIpsecParameter.md (hunk 18 above)
Comment (Nilambari) [reply src/ResourceManager/Network/Commands.Network/help/Set-AzureRmVpnClientIpsecParameter.md]: Done

File: src/ResourceManager/Dns/Commands.Dns.Test/ScenarioTests/ZoneTests.cs (hunk 2 above)
Comment (Nilambari) [reply src/ResourceManager/Dns/Commands.Dns.Test/ScenarioTests/ZoneTests.cs]: Enabled skipped tests back as tests are passing now.

**Comment by Nilambari** [comment]
@cormacpayne  Incorporated requested review comments. Please take a look and help accept the PR. Thanks!

**Comment by Nilambari** [comment]
@cormacpayne  I think build failed due to static analysis errors. It asks for adding ShouldProcess to commandlets:- New-AzureRmVpnClientIpsecPolicy & New-AzureRmVpnClientIpsecParameter which was removed as per latest code review comments. Can you Please advice what to do in this case in order for build to pass?

**Comment by Nilambari** [comment]
@cormacpayne Build is passed, Can you help accept the PR? Thanks!

**Comment by cormacpayne** [comment]
On-demand: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/847/

**Comment by cormacpayne** [comment]
On-demand: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/852/

**Comment by maddieclayton** [comment]
on demand here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/861/

**Comment by cormacpayne** [comment]
On-demand: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/868/

**Comment by Nilambari** [comment]
@cormacpayne  Seems on demand job is successful, can you Please help merge this PR?

**Pull request merged**